en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,453 @@
+   
+   <img src="../../../../assets/img/connectors/redis-payloadfactory.png" title="Add payloadfactory to extract IBM details" width="500" alt="Add payloadfactory to extract IBM details"/> 
+
+4. Initialize the connector.
+    
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Redis Connector** section. Then drag and drop the `init` operation into the Design pane.
+        
+        <img src="../../../../assets/img/connectors/redis-init-drag-and-shop.png" title="Drag and drop init operation" width="500" alt="Drag and drop init operation"/>   
+    
+    2. Add the property values into the `init` operation as shown below. Replace the `redisHost`, `redisPort`, `redisTimeout` with your values.
+        
+        - **redisHost** : The Redis host name (default localhost).. 
+        - **redisPort** : The port on which the Redis server is running (The default port is 6379).
+        - **redisTimeout** : The server TTL (Time to live) in milliseconds.
+    
+        <img src="../../../../assets/img/connectors/redis-init-parameterspng.png" title="Add values to the init operation" width="800" alt="Add values to the init operation"/>
+
+5. Set up the iPush operation. In this operation we can inserts one or more elements to the head of a list.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Redis Connector** section. Then drag and drop the `iPush` operation into the Design pane.
+           
+        <img src="../../../../assets/img/connectors/redis-ipush-operation.png" title="Drag and drop ipush operation" width="500" alt="Drag and drop ipush operation"/>    
+
+    2. In this operation we are going to inserts one or more elements to the head of a list.The lPush operation inserts one or more elements to the head of a list that is stored in a specified key.                                                                                          
+          
+        - **redisKey** : The name of the key where the list is stored.
+        - **redisStrings** : One or more elements that you want to add to the head of the list.
+        
+        While invoking the API, the above two parameters values extracted from the response of SimpleStockQuoteService. Then insert as a input value.
+        
+        <img src="../../../../assets/img/connectors/redis-ipush-parameters.png" title="iPush parameters" width="500" alt="iPush parameters"/> 
+    
+6. To get the input values in to the `iPush` we can use the [property mediator](../../../mediators/property-Mediator). Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown bellow.
+    
+   > **Note**: That the properties should be add to the pallet before create the operation.
```

Comment:
create operation means 'iPush'?

Reply:
This should be change as "lPush". Updated.